src/microbe_stage/Microbe.cs

```diff
@@ -476,6 +483,18 @@ public void ResetOrganelleLayout()
         allOrganellesDivided = false;
     }
 
+    /// <summary>
+    ///   Focefully change all the organelles color in this microbe independent of the set species' color
+    /// </summary>
+    public void ForceChangeOrganellesColour(Color colour)
```

Comment:
I think rather than having a method like this, passing in a fully controlled Species class that has the organelle colour modified would be nice.
If performance with that is affected then a new "ApplyOrganelleColours` method could be added here (if there isn't one already) that can be used to apply the changes made to the Species object.